examples/ingest/pinecone/ingest.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Processes all the files from s3://utic-dev-tech-fixtures/small-pdf-set/,
+# embeds the processed documents, and writes to results to a Pinecone index.
+
+# Structured outputs are stored in s3-small-batch-output-to-pinecone/
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+
+# As an example we're using the s3 source connector,
+# however ingesting from any supported source connector is possible.
+PYTHONPATH=. ./unstructured/ingest/main.py \
+        s3 \
```

Comment:
Same comment about using a local connector for the source

Reply:
with [update example](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/f884123c1465d6b31031c426d69497db2847330a)